src/System.Globalization/tests/CultureInfo/CultureInfoClone.cs

```diff
@@ -25,5 +25,15 @@ public void Clone(CultureInfo culture)
             Assert.Equal(culture, clone);
             Assert.NotSame(clone, culture);
         }
+
+        [Fact]
+        public void TestCalendarAfterCloning()
+        {
+            CultureInfo ci = new CultureInfo("en-US");
+            Assert.True(ci.Calendar == ci.DateTimeFormat.Calendar);
```

Comment:
might be worth adding a comment (second arg to Assert.True) to be able to tell which assert failed in case of regression

Reply:
I'll add that. Thank you.